### PR TITLE
Grant all master privs to ecs_user

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
## Description

tl;dr - Secure migration to grant all privs to ecs_user

Currently the `ecs_user` in our AWS RDS instance can't modify any tables. This makes it useless for using to do migrations which is the point of having this user. This change ensure we have given all privs that the master user has to the `ecs_user`. In production this same migration will read:

```sql
GRANT master TO ecs_user;
```

Locally the user is `postgres` but in RDS we use `master`.

## Setup

This is a little tricky to test so here's the way to do it locally:

```sh
# move this migration up in the queue
mv local_migrations/20190726151515_grant_all_ecs_user.up.sql local_migrations/20190129115417_grant_all_ecs_user.up.sql

echo "exec(\"./apply-secure-migration.sh 20190129115417_grant_all_ecs_user.up.sql\")" > migrations/20190726151515_grant_all_ecs_user.up.fizz

mv migrations/20190726151515_grant_all_ecs_user.up.fizz migrations/20190129115417_grant_all_ecs_user.up.fizz

update-migrations-manifest

# Make a new manifest
head -n222 migrations_manifest.txt > migrations_manifest_new.txt

# reset the DB
make db_dev_reset

# Migrations won't work outside this directory
cd scripts

# Use `postgres` user to do initial migrations up to and including the GRANT
milmove migrate -p ../migrations -m ../migrations_manifest_new.txt

# Complete remaining migrations using the `ecs_user` to prove it works
milmove migrate -p ../migrations -m ../migrations_manifest.txt --db-user ecs_user

# Then put things back in order
git reset --hard HEAD
```

To prove that it doesn't work without the `GRANT`:

```sh
# Make a new manifest
head -n222 migrations_manifest.txt > migrations_manifest_new.txt

# reset the DB
make db_dev_reset

# Migrations won't work outside this directory
cd scripts

# Use `postgres` user to do initial migrations up to and including the GRANT
milmove migrate -p ../migrations -m ../migrations_manifest_new.txt

# Complete remaining migrations using the `ecs_user` to prove it works
milmove migrate -p ../migrations -m ../migrations_manifest.txt --db-user ecs_user
```

You'll see an error very similar to what we see on RDS:

```
pg_dump: [archiver (db)] query failed: ERROR:  permission denied for relation shipment_line_item_dimensions
pg_dump: [archiver (db)] query was: LOCK TABLE public.shipment_line_item_dimensions IN ACCESS SHARE MODE
[POP] 2019/07/26 23:01:07 warn - Migrator: unable to dump schema: exit status 1
panic: error executing ../migrations/20190207001056_create_distance_calculations.up.fizz, sql: CREATE TABLE "distance_calculations" (
"id" UUID NOT NULL,
PRIMARY KEY("id"),
"origin_address_id" UUID NOT NULL,
"destination_address_id" UUID NOT NULL,
"distance_miles" integer NOT NULL,
"created_at" timestamp NOT NULL,
"updated_at" timestamp NOT NULL,
FOREIGN KEY ("origin_address_id") REFERENCES "addresses" ("id"),
FOREIGN KEY ("destination_address_id") REFERENCES "addresses" ("id")
);
ALTER TABLE "shipments" ADD COLUMN "shipping_distance_id" UUID;
ALTER TABLE "shipments" ADD CONSTRAINT "shipments_distance_calculations_id_fk" FOREIGN KEY ("shipping_distance_id") REFERENCES "distance_calculations" ("id");: pq: must be owner of relation shipments

goroutine 1 [running]:
main.main()
        /Users/cgilmer/Projects/transcom/mymove/cmd/milmove/main.go:109 +0xa42
```

## Code Review Verification Steps

* [ ] Tested in the Experimental Environment
* [x] Request review from a member of a different team.

### CODE REVIEW

File: pkg/migrate/find_migrations.go [hunk 1]
```diff
@@ -30,8 +28,9 @@ func FindMigrations(fm *pop.FileMigrator, valid map[string]struct{}, runner func
 				return nil
 			}
 
+			// Ignore files not in the manifest
 			if _, ok := valid[filepath.Base(p)]; !ok {
-				return errors.New(fmt.Sprintf("migration at path %q missing from manifest", p))
+				return nil
```
Comment (chrisgilmerproj) [inline pkg/migrate/find_migrations.go]: Should we log some kind of warning here?

Comment (mr337) [reply pkg/migrate/find_migrations.go]: I'm on the fence about this. The manifest is inclusion by default, and possible order, so the migration tool shouldn't care a single bit if there are extra files.

From a user perspective the worst thing is to add a migration file and forget to include it.  Whether to present that as stderror or just info as stdout is still up for discussion.

File: .circleci/config.yml [hunk 2]
```diff
@@ -815,33 +815,33 @@ workflows:
             - pre_deps_yarn
             - acceptance_tests_local
           # if testing on experimental, you can disable these tests by using the commented block below.
-          # filters:
```
Comment (mr337) [inline .circleci/config.yml]: For the build steps, what looks like a deploy, I believe I see the logic behind excluding the branch. For the tests any reason?

Comment (chrisgilmerproj) [reply .circleci/config.yml]: The tests take up to 30 minutes. For experimental they don't have to pass because we're just pushing from a branch. This allows us to deploy faster without waiting on tests that we don't care about.

File: pkg/migrate/find_migrations.go (hunk 1 above)
Comment (chrisgilmerproj) [reply pkg/migrate/find_migrations.go]: I think I'll add the error as a log line and let the user sort it out.

File: .circleci/config.yml (hunk 2 above)
Comment (mr337) [reply .circleci/config.yml]: Makes sense!

**Comment by chrisgilmerproj** [comment]
I've run this in experimental and the DB migration appears to have gone through. I'll test the efficacy with https://github.com/transcom/mymove/pull/2349.
